cmd/consensus/main.go

```diff
@@ -639,69 +642,6 @@ func main() {
 			}
 			return nil
 		}).
-		Component("hotstuff participant", func(_ cmd.NodeBuilder, node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
```

Comment:
Sorry, I don't understand why you are removing this. Unfortunately, the link doesn't provide much insight for me, because it just brings me to diff page with 300 changed files without any further indication. 

In the present version of the code, https://github.com/onflow/flow-go/blob/227479ef25a7a763ca667cb462d5c3a3f1a337ca/engine/consensus/compliance/engine.go#L160 is not called at all anymore. Based on my understanding, this code should not be deleted ... not sure what I am missing.

Reply:
Good catch. 

There was some conflicts on the [NewEpochAwareSignerStore](https://github.com/onflow/flow-go/blob/master/cmd/consensus/main.go#L617), which should be removed, but there is some change should be kept.